app/views/logs/_filter_logs_select.html.erb

```diff
@@ -1,12 +1,32 @@
-<div class="govuk-grid-column-two-thirds">
-  <%= form_with model: log_search_form, url: logs_path, method: :get do |form| %>
-    <%= form.hidden_field :filter_option %>
-    <%= form.hidden_field :username %>
-    <%= form.hidden_field :ip %>
-    <%= form.hidden_field :location_id %>
-    <%= form.govuk_select :success,
-                          options_for_select([["All", nil], %w[Successful true], %w[Failed false]], log_search_form.success),
-                          label: { text: "Status type:" } %>
-    <%= form.govuk_submit "Filter" %>
-  <% end %>
+<div class="govuk-grid-row govuk-!-padding-bottom-6">
+  <div class="govuk-grid-column-full">
+    <%= form_with model: log_search_form, url: logs_path, method: :get do |form| %>
+      <%= form.hidden_field :filter_option %>
+      <%= form.hidden_field :username %>
+      <%= form.hidden_field :ip %>
+      <%= form.hidden_field :location_id %>
+      <div class="govuk-cookie-banner">
```

Comment:
Maybe another way to make this box gray would be better? This is not a cookie banner and may be a bit confusing